web_programming/currency_converter.py

```diff
@@ -10,8 +10,9 @@
 URL_BASE = "https://www.amdoren.com/api/currency.php"
 TESTING = os.getenv("CI", False)
 API_KEY = os.getenv("AMDOREN_API_KEY", "")
+
 if not API_KEY and not TESTING:
-    raise KeyError("Please put your API key in an environment variable.")
+    raise KeyError("API key must be set as 'AMDOREN_API_KEY' in environment variables")
```

Comment:
I feel like the message is ambiguous.
```suggestion
    raise KeyError("API key must be provided with 'AMDOREN_API_KEY' environment variable.")
```